zonemaster/zonemaster-gui
# Information banner

## Purpose

This adds a banner that can be used to display a custom message to the users.
FYI I'm no expert with colors, I choose one I felt was fine to my eyes.

![zm-gui-info-1](https://user-images.githubusercontent.com/70589067/137725818-a5f9ea5b-31e6-43c9-bb1a-ee0831cd224f.png)

![zm-gui-info-2](https://user-images.githubusercontent.com/70589067/137725814-7a696d8e-d5af-408e-a90b-fed7e8946c8b.png)


## Context

N/A

## Changes

* Update the CSS to make the main component just below the navbar

* Add a new configuration item `msgBanner` that can be used to display a custom message on top of the page below the navbar

## How to test this PR

Set the `msgBanner` configuration item with some text. Reload the page, a banner with the text should be displayed. Removing the item from the configuration or setting an empty string should remove the banner.
